tsconfig-cjs.json

```diff
@@ -2,6 +2,7 @@
   "extends": "./tsconfig.json",
   "compilerOptions": {
     "module": "CommonJS",
-    "outDir": "dist/cjs"
+    "outDir": "dist/cjs",
+    "declaration": false
```

Comment:
we should test that in the wild, but imo having the declarations once (in the esm dir) should be enough.